master/reference/kube-controllers/configuration.md

```diff
@@ -1,17 +1,24 @@
 ---
-title: Configuring the Calico policy controller
+title: Configuring the Calico Kubernetes controllers
 ---
 
-The policy controller is primarily configured through environment variables. When running
-the policy controller as a Kubernetes pod, this is accomplished through the pod manifest `env`
+The Kubernetes controllers are primarily configured through environment variables. When running
+the controllers as a Kubernetes pod, this is accomplished through the pod manifest `env`
 section.
 
-## The calico/kube-policy-controller container
+## The calico/kube-controllers container
+
+The calico/kube-controllers container includes three controllers:
```

Comment:
Should we put `calico/kube-controllers` in code ticks? I raised a comment on the style guide PR pertaining to this: https://github.com/projectcalico/calico/pull/1137#issuecomment-332675118

Reply:
Not sure, @emanic?

Reply:
Yes, we really should I think. Thanks @ozdanborne